From 6a7800c00752dc0e68939d640274b9813cf0604e Mon Sep 17 00:00:00 2001
From: Alexander Egorenkov <egorenar@linux.ibm.com>
Date: Tue, 20 Oct 2020 12:55:44 +0200
Subject: [PATCH] genprotimg/boot: disable SSP

SSP cannot work with boot loaders because it requires libc.

Signed-off-by: Alexander Egorenkov <egorenar@linux.ibm.com>
---
 genprotimg/boot/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/genprotimg/boot/Makefile b/genprotimg/boot/Makefile
index d39bbd66..f2bc25c3 100644
--- a/genprotimg/boot/Makefile
+++ b/genprotimg/boot/Makefile
@@ -12,7 +12,7 @@ ALL_CFLAGS := $(NO_PIE_CFLAGS) -Os -g \
 	-DENABLE_SCLP_ASCII=1 \
 	-DS390_TOOLS_RELEASE=$(S390_TOOLS_RELEASE) \
 	-fno-builtin -ffreestanding -fno-asynchronous-unwind-tables \
-	-fno-delete-null-pointer-checks \
+	-fno-delete-null-pointer-checks -fno-stack-protector \
 	-fexec-charset=IBM1047 -m64 -mpacked-stack \
 	-mstack-size=4096 -mstack-guard=128 -msoft-float \
 	-Wall -Wformat-security -Wextra -Werror
-- 
2.26.2

